www/assets/js/components/SearchFilter.tsx

```diff
@@ -10,20 +10,20 @@ export default function SearchFilter(props: Props) {
   const { value, clearFacet, labelFunction } = props
 
   return (
-    <div className="active-search-filter">
+    <div className="active-search-filter" aria-label="active search filters">
```

Comment:
I thought this aria-label would be ignored because MDN says
> Not all elements can be given an accessible name. Neither aria-label nor aria-labelledby should be used with non-interactive elements

But VoiceOver does read it 🤷 That said, I'm not sure it's entirely necessary—there is "Filters" text right above it.

_I'm just rambling. Don't feel the need to change anything if you think it's helpful._

Reply:
You're right. I misunderstood the UI. I thought all the filters appeared in one bubble. The label doesn't make sense for a single filter. I'll remove it.

Additionally, maybe we can call the "close" button "clear filter" instead.